crux-test/test/crux/api_test.clj

```diff
@@ -25,15 +21,23 @@
            org.eclipse.rdf4j.query.Binding))
 
 (def api-implementations
-  (-> {:local-standalone (t/join-fixtures [fs/with-standalone-node kvf/with-kv-dir fapi/with-node])
-       :remote (t/join-fixtures [fs/with-standalone-node kvf/with-kv-dir fh/with-http-server fapi/with-node fh/with-http-client])
-       :h2 (t/join-fixtures [#(fj/with-jdbc-node :h2 %) kvf/with-kv-dir fapi/with-node])
-       :sqlite (t/join-fixtures [#(fj/with-jdbc-node :sqlite %) kvf/with-kv-dir fapi/with-node])
-       :local-kafka (-> (t/join-fixtures [fk/with-cluster-node-opts kvf/with-kv-dir fapi/with-node])
+  (-> {:local-standalone (t/join-fixtures [fix/with-standalone-topology fix/with-kv-dir fix/with-node])
+       :remote (t/join-fixtures [fix/with-standalone-topology
+                                 fix/with-kv-dir
+                                 fh/with-http-server
+                                 fix/with-node
+                                 fh/with-http-client])
+       :h2 (t/join-fixtures [#(fj/with-jdbc-node :h2 %) fix/with-kv-dir fix/with-node])
+       :sqlite (t/join-fixtures [#(fj/with-jdbc-node :sqlite %) fix/with-kv-dir fix/with-node])
+       :local-kafka (-> (t/join-fixtures [fk/with-cluster-node-opts fix/with-kv-dir fix/with-node])
                         (with-meta {::embedded-kafka? true}))
-       :kafka+remote-doc-store (-> (t/join-fixtures [fk/with-cluster-node-opts fs/with-standalone-doc-store kvf/with-kv-dir fapi/with-node])
+       :kafka+remote-doc-store (-> (t/join-fixtures [fk/with-cluster-node-opts
```

Comment:
In the future, I'd like to consider the api-test being a fixture and to be used by the kafka module, therefore moving all kafka related stuff out of `crux-test` into `crux-kafka`.

Reply:
agreed :)